README.md

```diff
@@ -55,4 +64,7 @@ forge script script/deploy/DeployYieldDistributor.s.sol:DeployYieldDistributor -
 1. Amend the `data` variable in `script/upgrades/UpgradeYieldDistributor.s.sol` to match desired data 
 2. run `forge clean && forge build && forge script script/upgrades/UpgradeYieldDistributor.s.sol --sig "run(address)" <proxy_address> --rpc-url $RPC_URL  --sender <proxy_admin>` 
 
-The proxy admin address is configured to be the Breadchain multisig at address `0x918dEf5d593F46735f74F9E2B280Fe51AF3A99ad` and the Yield Distributor proxy address is `0xeE95A62b749d8a2520E0128D9b3aCa241269024b`
\ No newline at end of file
+The proxy admin address is configured to be the Breadchain multisig at address `0x918dEf5d593F46735f74F9E2B280Fe51AF3A99ad` and the Yield Distributor proxy address is `0xeE95A62b749d8a2520E0128D9b3aCa241269024b`
```

Comment:
I'd remove this line. First part describes contract state which may change. Second part is deployment info. If anything this should be split into two sections. The multisig address can be added to general info at the top of the README. Then add another section, maybe in table form, that lists production deployments of the contracts contained in the repo, like:

| Contract | Address |
| ------------- | ------------- |
| Bread | `0xa555d5344f6FB6c65da19e403Cb4c1eC4a1a5Ee3` |
| YieldDistributor | `0xeE95A62b749d8a2520E0128D9b3aCa241269024b` |